pkg/execution/realtime/broadcaster_redis.go

```diff
@@ -75,6 +75,27 @@ func (b *redisBroadcaster) Publish(ctx context.Context, m Message) {
 	}
 }
 
+func (b *redisBroadcaster) PublishStream(ctx context.Context, m Message, data string) {
+	for _, t := range m.Topics() {
+		go func(t Topic) {
```

Comment:
Seeing as this might run for a little while with the retries, shall we also check for context cancellation?